content/applications/sales/crm/pipeline/merge_similar.rst

```diff
@@ -0,0 +1,113 @@
+=====================================
+Merge similar leads and opportunities
+=====================================
+
+Odoo automatically detects similar *leads* and *opportunities* within the *CRM* app. Identifying
+these duplicated records allows them to be merged without losing any information in the process.
+Not only does this help keep the *pipeline* organized, but it also prevents customers from being
+contacted by more than one salesperson.
+
+.. note::
+   When merging opportunities, no information is lost. Data from the other opportunity is logged in
+   the chatter, and the information fields, for reference.
+
+Identifying similar leads and opportunities
+===========================================
+
+Similar leads and opportunities are identified by comparing the *email address* and *phone number*
+of the associated contact. If a similar lead/opportunity is found, a *Similar Lead* smart button
+appears at the top of the lead or opportunity record.
+
+.. image:: merge_similar/similar-smart-button.png
+   :align: center
+   :alt: An opportunity record with emphasis on the Similar Leads smart button.
+
+Comparing similar leads and opportunities
+-----------------------------------------
+
+To compare the details of similar leads/opportunities, navigate to :menuselection:`CRM app -->
+Pipeline` or :menuselection:`CRM app --> Leads`. Open a lead or opportunity, and click the
+:guilabel:`Similar Leads` button at the top of the page. This opens a Kanban view with only similar
+leads/opportunities. Click on a card to view the details for the lead/opportunity, and confirm if
+they should be merged.
+
+Merging similar leads and opportunities
+=======================================
+
+.. important::
+   When merging, Odoo gives priority to whichever lead/opportunity was created in the system first,
+   merging the information into the first created lead/opportunity. However, if a lead and an
+   opportunity are being merged, the resulting record is referred to as an opportunity, regardless
+   of which record was created first.
+
+After confirming that the leads/opportunities should be merged, return to the Kanban view using
+breadcrumbs, or by clicking the :guilabel:`Similar Lead` smart button. Click the :guilabel:`≣ (view
+list)` icon to change to list view.
+
+Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
+:guilabel:`⚙️ Actions (gear)` icon at the top of the page, to reveal a drop-down menu. From that
+drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
+
+When :guilabel:`Merge` is selected from the :guilabel:`⚙️ Actions (gear)` drop-down menu, a
+:guilabel:`Merge` pop-up modal appears. In that pop-up modal, under the :guilabel:`Assign
```

Comment:
when it comes to "pop-up modal" vs. "pop-up window/pop-up form" -- we've actually been having a discussion about this recently. While "modal" is technically accurate, and how it shows up in the 'Developer Tools' -- I'm so sure that the average user/reader is aware of such a term, and is likely more accustomed to seeing "window/form." We haven't landed on anything *concrete* as of yet, so if you feel most comfortable with "modal," feel free to keep using it. But, if you're aiming to deliver a more approachable terminology, my opinion/suggestion would be to go with a more user-friendly "pop-up window/form" route. But, again, totally your call -- just figured I'd throw it out there for you to consider.